### PR TITLE
Videos Show Action and Tests

### PR DESCRIPTION
**Pull request by theomoondev**
I decided to rename the VIDEO_FIELDS constant at the top of the videos test since the expected fields are different for each action. There's one for VIDEO_INDEX_FIELDS and one for VIDEO_SHOW_FIELDS. Feel free to make suggestions on that!

### CODE REVIEW

File: test/test_helper.rb [hunk 1]
```diff
@@ -18,4 +18,12 @@ class ActiveSupport::TestCase
   fixtures :all
 
   # Add more helper methods to be used by all tests here...
+  def check_response(expected_type:, expected_status: :success)
```
Comment (theomoondev) [inline test/test_helper.rb:21]: What do you think about putting the check_response method in test_helper like this?

Comment (dnguye2) [reply test/test_helper.rb:21]: Yes I approve of this and it would make things cleaner

File: test/controllers/videos_controller_test.rb [hunk 2]
```diff
@@ -1,37 +1,56 @@
 require "test_helper"
 
 describe VideosController do
-  VIDEO_FIELDS = ["id", "title", "release_date", "available_inventory"].sort
+  VIDEO_INDEX_FIELDS = ["id", "title", "release_date", "available_inventory"].sort
+  VIDEO_SHOW_FIELDS = ["title", "overview", "release_date", "total_inventory", "available_inventory"].sort
```
Comment (dnguye2) [inline test/controllers/videos_controller_test.rb:5]: I really like how you separated index and show fields for this. Good catch!

**Review by dnguye2 — APPROVED** [review]
Overall looks good to me!

**Pull request merged**